.github/workflows/checks.yml

```diff
@@ -117,7 +117,7 @@ jobs:
 
       - name: Publish artifacts locally
         run: |-
-          sbt "publishLocal; publishM2"
+          sbt "++2.13; publishLocal; publishM2"
```

Comment:
this shouldn't be needed, we have 2.13 as the default scalaVersion

Reply:
I switched that to 2.12 to catch using newer features early